targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_LPC/qspi_api.c

```diff
@@ -0,0 +1,353 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018, ARM Limited
```

Comment:
HAL implementation - apache licenses (other files have), only drivers are bsd 3 licences, can you update this file then? + adding SDPX identifier

Reply:
I have made the change